docs/_includes/head/custom.html

```diff
@@ -0,0 +1,5 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
+<link rel="manifest" href="/assets/images/favicon/site.webmanifest">
```

Comment:
Why do we need a manifest config for the favicon?

Could we just put a multi-sized favicon.ico file in the webroot to solve this problem?

Reply:
I am afraid not. I have not found a proper place to put the favicon link. However, I find this solution [in this issue](https://github.com/mmistakes/minimal-mistakes/issues/949) that custom header via `_includes`.

Reply:
OK, let's try to use this solution.

P.S. It would be great if we can use a single favicon.ico file and just put it to the web root directory.